### PR TITLE
change title on README

### PR DESCRIPTION
**Pull request by UP941831**

### CODE REVIEW

**Comment by UP941831** [comment]
bla bla bla

**Comment by UP941831** [comment]
maybe i do this

**Review by UP941831 — COMMENTED** [review]
Nice title bro.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
```
Comment (UP941831) [inline README.md:1]: beutiful amazing

Comment (UP941831) [inline README.md:1]: wat

**Review by butjar — APPROVED** [review]
Lgtm

**Review by UP941831 — COMMENTED** [review]
boom

**Pull request closed without merging**